spec/models/expense_spec.rb

```diff
@@ -7,9 +7,28 @@
     it { is_expected.to validate_presence_of(:name) }
     it { is_expected.to validate_uniqueness_of(:name) }
     it { is_expected.to validate_presence_of(:frequency) }
+    it { is_expected.to define_enum_for(:frequency).with(Expense.frequencies) }
   end
 
   context "associations" do
     it { is_expected.to belong_to(:budget) }
   end
+
+  context "frequency has a default value" do
+    it "should be weekly" do
+      expense = create(:expense)
```

Comment:
Instead of creating a record in the database, can we build the record?

Reply:
Done in sha: 0132e1f